src/android/app/build.gradle.kts

```
@@ -150,7 +150,7 @@ android {
 
     externalNativeBuild {
         cmake {
-            version = "3.22.1"
+            version = "3.30.3"
```

I looked into it, and this value does have syntax to request a minimum CMake version rather than an exact version. We can do so like this:
```suggestion
            version = "3.25.0+"
```
The version you specified in build.yml, 3.30.3, will still be used in CI as it is the only CMake version in that environment which fulfils this requirement. This mostly just makes building locally less annoying.